pwa/src/templates/templateParts/sourcesForm/SourceFormTemplate.tsx

```diff
@@ -252,25 +252,21 @@ export const SourceFormTemplate: React.FC<SourceTemplateProps> = ({ source }) =>
                   <>
                     <FormField>
                       <FormFieldLabel>{t("Status")}</FormFieldLabel>
-                      <div className={clsx(styles[getStatusColor(source.status ?? "no known status")])}>
-                        <Tag
-                          icon={<FontAwesomeIcon icon={getStatusIcon(source.status ?? "no known status")} />}
-                          label={source.status?.toString() ?? "no known status"}
-                        />
-                      </div>
+
+                      <div>{getStatusTag(source.status)}</div>
                     </FormField>
 
                     <FormField>
                       <FormFieldInput>
                         <FormFieldLabel>{t("Created")}</FormFieldLabel>
-                        <Tag label={translateDate(i18n.language, source.dateCreated) ?? "-"} />
+                        <StatusTag label={translateDate(i18n.language, source.dateCreated) ?? "-"} />
```

Comment:
I think that we need a gap between the tag and label for Created and Modified or that we need to put the tag underneath the label 
![image](https://user-images.githubusercontent.com/43807324/220605776-d5336413-f25d-4dc0-95cb-3f86213a41bf.png)